py/sight/requirements-worker.txt

```diff
@@ -1,4 +1,5 @@
 absl-py
+bayesian-optimization==1.4.3
```

Comment:
This is a requirement for just the optimizer worker, so it belongs in a requirements.txt file for that worker rather than the general file for all workers